pkg/cli/admin/release/info.go

```diff
@@ -736,15 +736,23 @@ func (i *ReleaseInfo) PreferredName() string {
 }
 
 func (i *ReleaseInfo) Platform() string {
-	os := i.Config.OS
-	if len(os) > 0 {
```

Comment:
This wasn't causing trouble before, because `describeReleaseInfo` wasn't using this `Platform` method, but the `>` was a bug, and I'm fixing it to `==`.